src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -234,7 +235,8 @@ const useCoreChatSystemHook = () => {
     }
 
     refresh('channelList');
-    refresh('chat');
+    refreshWithId('chat', websocketData?.channel?.id);
+    // refresh('chat');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `refresh('chat')` to `refreshWithId('chat', websocketData?.channel?.id)` seems to be a good improvement as it allows refreshing data with a specific ID. However, please ensure that `websocketData?.channel?.id` always provides a valid id. If there's a chance that it could be `undefined` or `null`, you should handle this case to prevent potential issues.


<!-- This is an auto-generated comment by OSS CodeRabbit -->